README.md

```diff
@@ -60,7 +60,7 @@ yarn test
 To do it programmatically, see the command below, but use your own `NETWORK` and `DAPI_NAME` values.
 
 ```sh
-NETWORK=polygon-testnet DAPI_NAME=ETH/USD yarn deploy-dapi-proxy
+NETWORK=ethereum-sepolia-testnet DAPI_NAME=BTC/USD yarn deploy-dapi-proxy
```

Comment:
The data feed ID below belongs to ETH/USD so you should also change that if you will change this to BTC/USD